src/components/App.js

```diff
@@ -0,0 +1,22 @@
+import ButtonPanel from './ButtonPanel';
+import Display from './Display';
+
+function App() {
```

Comment:
- [ ] [OPTIONAL] It will be nice if you use the arrow function as you are using ES6 syntax